examples/simple_autopilot_public/main.tf

```diff
@@ -33,8 +33,8 @@ provider "kubernetes" {
 }
 
 module "gke" {
-  source  = "terraform-google-modules/kubernetes-engine/google//modules/beta-autopilot-public-cluster"
-  version = "~> 33.0"
+  source = "../../modules/beta-autopilot-public-cluster"
+  #version = "~> 33.0"
```

Comment:
assume this was updated and checked in by accident?

Reply:
haha, yes